src/lib/viewers/media/Dash.scss

```diff
@@ -1,12 +1,17 @@
 @import 'mediaBase';
 
+.bp-media-container {
+    display: flex;
```

Comment:
Are the flex container and flex styles needed? It looks like everything is absolutely positioned inside the .bp-media-container besides the actual media element

Reply:
It was necessary to center the media element, if I remember correctly. Will verify.

Reply:
Yeah, it's used to center the `video` element within the container.